dist/tools/pyterm/pyterm

```diff
@@ -125,8 +125,9 @@ class SerCmd(cmd.Cmd):
         self.ignores = []
         self.json_regs = dict()
         self.init_cmd = []
+        self.fmt_str = None
```

Comment:
This won't work, because `load_config()` set only parameters that are not yet existing (in order to not overwrite command line settings).